index.html

```diff
@@ -0,0 +1,183 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Capstone</title>
+    <link rel="stylesheet" href="style.css">
+    <script
+      src="https://kit.fontawesome.com/5d13e94c8b.js"
+      crossorigin="anonymous"
+    ></script>
+</head>
+<body>
+    <header class="header">
+        <div class="header-div">
+            <div>
+            <ul class="icons-ul">                
+                <li><img class="head-icon" src="./images/faceb.png" alt="facebookicon"></li>
+                <li><img class="head-icon" src="./images/twtr.png" alt="twitter-icon"></li>
+                <li class="link">Other language</li>
+                <li class="link">my page</li>
+                <li class="link">Logout</li>
+            </ul>
+            </div>
+            
+            <div class="bars">
+        </div>
+                 <nav>
+                    <div class="span-"><img  src="./images/download.jpg" alt="#" class="inner-backg"><p class="logo">LIVING FAITH CHURCH</p> </div>
+
+                    <ul class="nav-ul">
+                        <li><a href="about.html">About</a></li>
+                        <li><a href="index.html">Home</a></li>
+                        <li>Program</li>
+                        <li>Join</li>
+                        <li>Sponsor</li>
+                        <li>News</li>
+                        <li><h3 class="winners">Winners Chapel</h3></li>
+                    </ul>
+                </nav>
+            </div>
+    </header>
+    
+    <section>
+        <article class="intro">
+            <div class="menu-nav">
+                <div class="hamburger"></div>
+                <div class="hamburger"></div>
+                <div class="hamburger"></div>
+          </div>
+
+           
+<div class="mobileMenu"> 
+  
+<div class="m1"><h2 class="m2">Winners Chapel</h2></div>
+
+<span class="closeMenu">&times;</span>
+  <ul class="container">
+        <li><a href="about.html">About</a></li>
+        <li><a href="index.html">Home</a></li>
+        <li>Program</li>
+        <li>Join</li>
+        <li>News</li>
+ </ul>
+
+</div>
+        </article>
+
+    </section>
+
+    <main>
+        
+<div class="pg1">
+    <h2 class="theme">
+        "Youth For Christ"
+     </h2>
+     <h1 class="ch-name">Winners Chapel Youth Conference 2022.</h1>
+    <div class="moment">
+         <p class="prg">in His presnce there is fullness of joy, experience God in another demention come youth 2022 conference</p>
+    </div>
+     <div class="info">
+        <h6 class="date">22.05.2022(THUR)~23(FRI)</h6>
+        <p class="venue">@ Church Auditorium, Lagos.</p>
+    </div>
+ </div>
+
+ <section class="prog-card">
```

Comment:
- [ ] Please have a look at your CSS and ensure a proper alignment for all the elements, if you use flex-box or grid, this could be done with `justify-elements: center`

![image](https://user-images.githubusercontent.com/52635378/164944996-d1371cd2-47ed-424a-a589-030effffac6f.png)

Reply:
fixed